packages/react-topology/package.json

```diff
@@ -32,14 +32,28 @@
     "@patternfly/react-core": "^4.9.3",
     "@patternfly/react-icons": "^4.2.3",
     "@patternfly/react-styles": "^4.2.2",
-    "tslib": "^1.11.1"
+    "tslib": "^1.11.1",
+    "mobx": "^5.14.2",
```

Comment:
Why do we need mobx?

Reply:
At the moment functionality is heavily dependent on these @observables that automatically updates and rerenders graph when something changes.